docs/XBlocks.md

```diff
@@ -0,0 +1,217 @@
+# Group Project building blocks
+
+The following blocks are a part of Group Project XBlock v2.
+
+## Top-level XBlocks
+
+* ` Group Project` - top level XBlock, represents entire group project as a whole
+    * Settings:
+        * `Display Name` - human-friendly name for Group Project - displayed in admin console.
```

Comment:
@itsjeyd "dashboard" have a very specific meaning in this project - it's one of the admin tools that displays students progress through the project. So, admin dashboard is a part of admin console.